Formula/etcher-cli.rb

```diff
@@ -0,0 +1,31 @@
+require "language/node"
+
+class EtcherCli < Formula
+  desc "Flash OS images to SD cards & USB drives, safely and easily"
+  homepage "https://etcher.io/"
+  url "https://github.com/resin-io/etcher.git",
+    :tag => "v1.4.4",
+    :revision => "cbd531e161a12721fb763b526781f66ef8237ac9"
+
+  depends_on "python" => :build
+  depends_on "jq"
+  depends_on "node"
```

Comment:
I don't think, that it will run against `node` 10 (at least anything else than a simple version check), because some native addons only compile against `node@6`.

Reply:
I created issues on etcher to allow building against system node, allow overridding the makefile node versions, and removing electron deps for the etcher-cli. Hopefully those will happen soon-ish, and then we won't need to maintain these patches.

Its so nice to have 2 minute build times vs the 30 minutes though, would totally be happy with just https://github.com/chrmoritz/homebrew-core/commit/f52479dd48229e8cd7df2ba1b465093ddfa715c6 making it into homebrew-core :)

As an aside, I think node v6 support is a feature so that the widest array of people and situations can use ether (not everyone can have the latest node installed, unfortunately).